en/core-libraries/collections.rst

```diff
@@ -337,52 +377,23 @@ function using the ``some()`` method::
         return $person->age < 21;
     });
 
-.. php:method:: match(array $conditions)
-
-If you need to extract a new collection containing only the elements that
-contain a given set of properties, you should use the ``match()`` method::
-
-    $collection = new Collection($comments);
-    $commentsFromMark = $collection->match(['user.name' => 'Mark']);
-
-.. php:method:: firstMatch(array $conditions)
-
-The property name can be a dot-separated path. You can traverse into nested
-entities and match the values they contain. When you only need the first
-matching element from a collection, you can use ``firstMatch()``::
-
-    $collection = new Collection($comments);
-    $comment = $collection->firstMatch([
-        'user.name' => 'Mark',
-        'active' => true
-    ]);
-
-As you can see from the above, both ``match()`` and ``firstMatch()`` allow you
-to provide multiple conditions to match on. In addition, the conditions can be
-for different paths, allowing you to express complex conditions to match
-against.
 
 Aggregation
 ===========
 
-.. php:method:: reduce(callable $c)
+.. php:method:: max(string|callable $callback, $type = SORT_NUMERIC)
 
-The counterpart of a ``map()`` operation is usually a ``reduce``. This
-function will help you build a single result out of all the elements in a
-collection::
+The same can be applied to the ``max()`` function, which will return a single
```

Comment:
With the new ordering, I'm not sure this sentence makes sense anymore. What is 'same' here now?